src/mongo/db/curop.h

```diff
@@ -527,6 +535,9 @@ class CurOp {
 
     std::string _planSummary;
 
+    // Pseudo RNG for rate limiter feature
+    static PseudoRandom _prng;
```

Comment:
This is an internal detail that is not a part of class's interface. It doesn't have any logical connection with the class that represents this command, except that it's used in one place to provide randomness. By putting it to the header, you introduce extra dependency on extra header file (random.h) which slows down compilation time for all the sources that include this header.